cluster/components/kuberay/ingress.yaml

```
@@ -0,0 +1,19 @@
+  ingressClassName: kong
+  rules:
+  - host: "k8s.local"
+    http:
+      paths:
+      - path: /raycluster
+        pathType: ImplementationSpecific
+        backend:
+          service:
+            name: rayservice-sample-head-svc
+            port:
+              number: 8265
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)